airbyte-integrations/connectors/source-fastbill/source_fastbill/components.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+import base64 
+from airbyte_cdk.sources.declarative.auth.token import BasicHttpAuthenticator
+from airbyte_cdk.sources.declarative.types import Config
+
+
+@dataclass
+class CustomAuthenticator(BasicHttpAuthenticator):
+
+    config: Config
+    @property
+    def token(self) -> str:
+
+        username = self.config['username'].encode("latin1")
+        password = self.config['api_key'].encode("latin1")
```

Comment:
Please check the `BasicHttpAuthenticator` implementation. You need to use interpolated strings here and in the function below.

Reply:
@marcosmarxm changed